wsapi.go

```diff
@@ -267,9 +267,16 @@ type helloOp struct {
 // FailedHeartbeatAcks is the Number of heartbeat intervals to wait until forcing a connection restart.
 const FailedHeartbeatAcks time.Duration = 5 * time.Millisecond
 
+// latency returns the latency between heartbeat acknowledgement and heartbeat send
```

Comment:
Perhaps we can name this HeartbeatLatency? Latency would lead me to believe this is the round trip/ping time.

Reply:
Sure, sounds good.